modules/ROOT/pages/access-client-app-id-task.adoc

```diff
@@ -1,17 +1,22 @@
-= Getting Client ID and Client Secret of a registered Client Application
+= Obtaining Client Credentials of a Registered Client Application
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-As a client application owner of a registered client, you might be required to provide client application credentials when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information for the client application. 
+As an owner of a xref:exchange::to-request-access.adoc[registered client application], you might be required to provide the client ID and the client secret of the application when accessing an API. After a contract is created between the client application and the API, you can access the API using these credentials based on the policies that are applied. For example, if the API owner has applied the Client ID Enforcement policy on the API, you must provide the client ID, and optionally, the client secret information for the client application. 
```

Comment:
Style guide says to avoid "might". Suggest "you are sometimes required"

Reply:
Done! Thanks!